### ISSUE TITLE
assoc_programm ctx.env.append_value('STRIPFLAGS') neither needed nor working (AIX)

### ISSUE THREAD

**Issue by aixtools** [pyinstaller/pyinstaller#4730]
## Context information (for bug reports)

* Output of `pyinstaller --version`: ```3.6```
* Version of Python: e.g. 3.7.4, 3.6.10
* Platform: AIX
* Did you also try this on another platform? Does it work there?
No, AIX specific

###  Details:
```
(py37) aixtools@x064:[/home/aixtools/PyInstaller-3.6]python setup.py build
running build
running build_bootloader
No precompiled bootloader found. Trying to compile it for you ...
Setting top to                           : /home/aixtools/PyInstaller-3.6/bootloader
Setting out to                           : /home/aixtools/PyInstaller-3.6/bootloader/build
Python Version                           : 3.7.4 (default, Dec 24 2019, 19:05:28) [C]
Checking for 'xlc' (C compiler)          : /usr/vac/bin/xlc_r
Checking size of pointer                 : 4
Platform                                 : AIX-32bit detected based on compiler
Checking for compiler flags -q32         : yes
Checking for linker flags -q32           : yes
Checking for library dl                  : yes
Checking for library m                   : yes
Checking for library z                   : yes
...
[12/13] Linking build/debug/run_d
[13/13] Processing build/debug/run_d
/usr/vac/bin/xlc_r: 1501-216 (W) command option -32 is not recognized - passed to ld
ld: 0706-012 The -X flag is not recognized.
ld: 0706-012 The -3 flag is not recognized.
ld: 0706-012 The -2 flag is not recognized.

Waf: Leaving directory `/home/aixtools/PyInstaller-3.6/bootloader/build/debug'
Build failed
 -> task in 'run_d' failed with exit status 255 (run with -v to display more information)
ERROR: Failed compiling the bootloader. Please compile manually and rerun setup.py

```

## Fix (submitted as PR - against develop, will need backport - please)
```
aixtools@x064:[/data/prj/python/git/pyinstaller]git diff
diff --git a/bootloader/wscript b/bootloader/wscript
index 50c297c4..69c15832 100644
--- a/bootloader/wscript
+++ b/bootloader/wscript
@@ -642,12 +642,6 @@ def configure(ctx):
         # This tool allows reducing the size of executables.
         ctx.find_program([assoc_programm(ctx, 'strip')], var='STRIP')
         ctx.load('strip', tooldir='tools')
-        # There is a strip flag for AIX environment
-        if ctx.env.DEST_OS == 'aix':
-            if ctx.env.PYI_ARCH == '32bit':
-                ctx.env.append_value('STRIPFLAGS', '-X32')
-            elif ctx.env.PYI_ARCH == '64bit':
-                ctx.env.append_value('STRIPFLAGS', '-X64')
```

## Comment
FYI - the correct way to accomplish this on AIX is to export the variable OBJECT_MODE=64 - for 64-bit builds, and either unset OBJECT_MODE, or set to 32 for 32-bit builds. 32-bit build is still the default mode for applications on AIX.

**Comment by Legorooj** [comment]
@aixtools can you link the PRs to your issues? With `Closes #issuenum` in the main comment.

**Comment by aixtools** [comment]
please excuse my ignorance - how do I do that?

p.s. I'll try to remember how to write a better title :)

**Comment by Legorooj** [comment]
@aixtools if you want to link issues, you have to - in the original comment - say Closes #4730 or whichever issue you want. There are other keywords, eg Fixes, that do this, but Closes is the most common.
